website/blog/Auth-with-Casbin-in-Apache-APISIX.md

```diff
@@ -0,0 +1,186 @@
+---
+title: "在 Apache APISIX 中使用 Casbin 进行授权"
+author: 作者：Casbin 社区 & Apache APISIX 社区
+---
+<!--truncate-->
+
+当我们在使用 Apache APISIX 时，可能想要在应用里添加复杂的授权逻辑。在此篇文章中，我们将使用 Apache APISIX 的内置 Casbin 插件（authz-casbin）来实现基于角色的访问控制（RBAC）模型。
+
+## 介绍
+
+### Apache APISIX
+
+[Apache APISIX](https://github.com/apache/apisix) 是一个动态、实时、高性能的 API 网关， 提供负载均衡、动态上游、灰度发布、精细化路由、限流限速、服务降级、服务熔断、身份认证、可观测性等数百项功能。你可以使用 Apache APISIX 来处理传统的南北向流量，以及服务间的东西向流量， 也可以当做 [k8s ingress controller](https://github.com/apache/apisix-ingress-controller) 来使用。
+
+### Casbin
+
+[Casbin](https://casbin.org/zh-CN/) 是一个强大的、高效的开源访问控制框架，其权限管理机制支持多种访问控制模型。
+
+### authz-casbin 插件介绍
+
+在 Apache APISIX 的使用中，路由匹配和请求授权之间有个隐含的矛盾点：为了更高细粒度的权限控制，需要配置更高细粒度的路由，来精准识别请求并对请求进行授权。在复杂的授权模型场景下，这将导致路由数量成倍增加，加剧了运维复杂度。
+authz-casbin 是一个基于 lua-casbin  的 Apache APISIX 插件，支持基于各种访问模型的强大授权。Casbin 是一个强大的、高效的开源访问控制框架，支持 ACL、RBAC、ABAC 等访问控制模型，lua-casbin 是 Lua 版本的授权访问模型。
+authz-casbin 插件可以把路由匹配和请求授权这两个功能很好地进行解耦，你可以将各种授权访问模型加载到 Apache APISIX 中，借助 lua-casbin 实现高效复杂的授权模式。
+
+**注意**：如果你想要实现身份验证（authentication），你需要使用其他插件或者自己来配置完成验证用户身份。
```

Comment:
how about  recommend available authentication plugins to users and add internal links to the plugins doc here

Reply:
Good idea. @tzssangglass Can you help me add links ?

Reply:
```suggestion
**注意**：如果你想要实现身份验证（authentication），你需要使用其他插件或者自己来配置完成验证用户身份，比如 [jwt-auth](https://github.com/apache/apisix/blob/master/docs/zh/latest/plugins/jwt-auth.md) 插件。
```